google-cloud-language/src/main/java/com/google/cloud/language/v1/PagedResponseWrappers.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.google.cloud.vision.spi.v1;
```

Comment:
This is git getting confused. Notice that the destination file is language here too. If you browse the final state of the code after my change, you'll see it's in order.